### PR TITLE
added link to example script for fetching trading account ids

### PR DESCRIPTION
**Pull request by kenho811-b1**

### CODE REVIEW

File: src/trading-api/next/v1/bullish-trading-api.yml [hunk 1]
```diff
@@ -532,6 +533,8 @@ info:
     ## Fetch Trading Account Ids
     
     Trading Account Ids may be required by some endpoints. They can be fetched from [Get All Trading Accounts](#get-/accounts/trading-accounts).
+    
+    An example script is available [here](https://github.com/bullish-exchange/api-examples/blob/master/get_trading_accounts.py).
```
Comment (kenho811-b1) [inline src/trading-api/next/v1/bullish-trading-api.yml:537]: Added example script.

image

File: src/trading-api/next/v2/bullish-trading-api.yml [hunk 2]
```diff
@@ -587,6 +588,8 @@ info:
     ## Fetch Trading Account Ids
     
     Trading Account Ids may be required by some endpoints. They can be fetched from [Get All Trading Accounts](#get-/accounts/trading-accounts).
+    
+    An example script is available [here](https://github.com/bullish-exchange/api-examples/blob/master/get_trading_accounts.py).
```
Comment (kenho811-b1) [inline src/trading-api/next/v2/bullish-trading-api.yml:592]: Added example script.

image

File: src/trading-api/next/v1/bullish-trading-api.yml [hunk 3]
```diff
@@ -190,8 +190,8 @@ info:
     - The [Get Asset By Symbol](#get-/assets/-symbol-) API endpoint for a specific asset symbol
     
     The precision of an asset is the number of decimal places used to express its value in.
-    For instance, the quantity in the market `BTCUSD` is in BTC, BTC has a precision of 8 and therefore the quantity in `BTCUSD` has 8 decimal places.
-    To express a quantity of 1 BTC in `BTCUSD` market, therefore, it has to be `1.00000000` BTC.
+    For instance, the quantity in the market `BTCUSD` is in BTC. As BTC has a precision of 8, the quantity in `BTCUSD` has 8 decimal places.
+    It follows that in order to express a quantity of 1 BTC in `BTCUSD` market, it has to be `1.00000000` BTC.
```
Comment (kenho811-b1) [inline src/trading-api/next/v1/bullish-trading-api.yml:194]: Correct **run-on sentence**

**Review by sandeeprakhrablockone — APPROVED** [review]
👌

**Pull request merged**